packages/esm-patient-common-lib/package.json

```diff
@@ -33,24 +28,11 @@
     "url": "https://github.com/openmrs/openmrs-esm-patient-chart/issues"
   },
   "dependencies": {
-    "@carbon/charts-react": "^0.41.14",
-    "@carbon/icons-react": "^10.18.0",
-    "@typescript-eslint/eslint-plugin": "^4.15.1",
-    "@typescript-eslint/parser": "^4.15.1",
-    "carbon-components-react": "^7.25.0",
-    "d3": "^5.16.0",
-    "lodash-es": "^4.17.15",
     "systemjs-webpack-interop": "^1.2.1"
```

Comment:
Do we still need this? I mean I don't see a webpack build for this one (and since we bundle it in we don't have an issue with paths anyway).

Reply:
I guess we don't need this either then:
https://github.com/openmrs/openmrs-esm-patient-chart/blob/feature/monorepo/packages/esm-patient-common-lib/src/set-public-path.ts